README

```diff
@@ -60,16 +69,28 @@ under the GPLv2; the original site has disappeared.
     :regexp "[^][()'\" \t\n]+"
     :ignore-case t
     :doc-spec '(("(ansicl)Symbol Index" nil nil nil)))
-     * Run make html if you want a HTML version. You can also produce
-       Docbook and XML versions with the xml and docbook targets.
+     * Run "FMT=foo make ansicl.foo" if you want a FOO version where
+       foo ∈ (docbook, html, epub3, latex, plaintext, xml, dvi,
+       dvipdf, ps, pdf).
 
   Problems
 
    Bugs in Texinfo and Emacs rather than the converter as such:
      * Backreferences from the list of figures are broken in Info.
      * makeinfo crashes on the XML, Docbook and HTML targets on Mac OS X.
+     * makeinfo (7.0.3) on OpenSuse Tumbleweed crashes on HTML and
+       Epub3 targets on with "realloc(): invalid next size".
      * Emacs on Mac OS X doesn't display all characters even though the
        font does contain them.
+   Bugs in dpans2texi.el:
+     * Emacs warns about lack of lexical-binding directive, I'm not
```

Comment:
I doubt it would break anything. If you wanted to do it you'd mostly have to go through and mark variables unused (prefixing them with _).